style.css

```diff
@@ -0,0 +1,124 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+
+body{
+  margin: 0;
+  /*background:linear-gradient(90deg, rgb(22, 22, 78), rgb(207, 207, 223), rgb(255, 255, 255), rgb(207, 207, 223), rgb(22, 22, 78));*/
+  background-color: whitesmoke;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  font-family: 'Lato', sans-serif; 
+
+  
```

Comment:
I suggest removing unused white spaces in the code. The code will be standardized and look cleaner